ListManagement/processOpenSlidesParticipants.py

```diff
@@ -0,0 +1,184 @@
+import argparse
+import sys
+import dataclasses
+import datetime
+import csv
+import logging
+import os
+
+class Constants:
+    WORKING_DIR = os.path.join(os.path.dirname(__file__), "workingDir")
+    LOG_NAME = f"process_openslides_list_{datetime.datetime.strftime(datetime.datetime.now(),'%Y_%m_%d_%H_%M_%S')}.txt"
+    LOG_PATH = os.path.join(WORKING_DIR,LOG_NAME)
+    class OpenSlidesCols:
+        TITLE = "title"
+        FIRST_NAME = "first_name"
+        LAST_NAME = "last_name"
+        EMAIL = "email"
+        PRONOUN = "pronoun"
+        GENDER = "gender"
+        USERNAME = "username"
+        DEFAULT_PASSWORD = "default_password"
+        IS_ACTIVE = "is_active"
+        IS_PHYSICAL_PERSON = "is_physical_person"
+        SAML_ID = "saml_id"
+        IS_PRESENT = "is_present"
+        STRUCTRE_LEVEL = "structure_level"
+        NUMBER = "number"
+        VOTE_WEIGTH = "vote_weight"
+        GROUPS = "groups"
+        COMMENT = "comment"
+        TOP_ROW = [TITLE, FIRST_NAME,LAST_NAME,EMAIL,PRONOUN,GENDER, USERNAME, DEFAULT_PASSWORD, IS_ACTIVE, IS_PHYSICAL_PERSON,SAML_ID,IS_PRESENT,STRUCTRE_LEVEL,NUMBER,VOTE_WEIGTH,GROUPS,COMMENT]
+    class ANCols:
+        FIRST_NAME = "first_name"
+        LAST_NAME = "last_name"
+        EMAIL = "email"
+        PRONOUNS = "pronouns"
+
+class CommmandFlags:
+    PREVIOUS_LIST = "-p"
+    PREVIOUS_LIST_LONG = "--prev-list"
+    MEMBERSHIP_LIST = "-m"
+    MEMBERSHIP_LIST_LONG = "--membership-list"
+    OUTPUT = "-o"
+    OUTPUT_LONG = "--output"
+    HELP = "-h"
+    HELP_LONG = "--help"
+    USAGE = "usage: python (python3 on mac) processOpenSlidesParticipants.py -p exportFromLastGBM.csv -m currentMembershipListFromActionNetwork.csv -o output-csv"
+
+    def __init__(self, prevListPath, membershipListPath, outputPath) -> None:
+        self.prevListPath = prevListPath
+        self.mebershipListPath = membershipListPath
+        self.outputPath = outputPath
+
+def setup():
+    if not os.path.exists(Constants.WORKING_DIR):
+        os.mkdir(Constants.WORKING_DIR)
+    logging.basicConfig(filename=Constants.LOG_PATH, level=logging.INFO, format="%(asctime)s : %(levelname)s : %(message)s")
+    logging.getLogger().addHandler(logging.StreamHandler(sys.stdout))
+    logging.info("Setup Complete")
+
+def parseArgs():
+    parser = argparse.ArgumentParser(description=CommmandFlags.USAGE)
```

Comment:
I'd probably just write this as a `help = ` statement, like the documentation: https://docs.djangoproject.com/en/5.0/howto/custom-management-commands/

But, I don't think this is a huge problem other than readability.